test/sql/test_time_fn/T/test_time_fn

```diff
@@ -197,6 +197,29 @@ select dayofweek_iso("2023-01-03");
 select dayofweek_iso("");
 select dayofweek_iso(NULL);
 
+-- name: test_date_diff
```

Comment:
plz add some test cases that the parameters contain NULL and the third paramter is not invalid

Reply:
i had add testcase contained NULL，and if third paramter is valid would report a error like the picture

Reply:
> i had add testcase contained NULL，and if third paramter is valid would report a error like the picture: mysql> select date_diff("2010-11-30 23:59:59", "2010-11-30 20:59:59", "aaa"); ERROR 1064 (HY000): Expr evaluate meet error: type column should be one of day/hour/minute/second/millisecond

should be added in test cases

Reply:
get，had add this to testcases